testimonials/mercury.yaml

```diff
@@ -3,4 +3,4 @@ companyName: Mercury
 logoURL: logos/mercury.png
 shortTestimonial: |
   Mercury is a bank for businesses. We use Haskell to meet our
-  customers high standards for correctness and security.
+  customers' high standards for correctness and security.
```

Comment:
Customer is already a plural / Noun , so you can not make it to customers or customer's . so change it to customer .

Reply:
ok i am gonna change it's to customer's